feat: add enable social auth locally docs (#687)
openedx/frontend-app-authn: docs/how_tos/enable_social_auth.rst

Enable Social Auth Locally
--------------------------

Please follow the steps below to enable social auth (SSO) locally.

1. Follow `Enabling Third Party Authentication <https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/tpa/index.html>`_ for backend configuration.

2. Authn has a component for rendering Social Auth providers at frontend which goes through each provider.

   * If the provider has an ``iconImage``, then it will be rendered as image in SSO button.

   * If ``iconImage`` is not available in provider, but the provider's ``iconClass`` is from the supported icon classes ``['apple', 'facebook', 'google', 'microsoft']`` then it is used as icon image.

>    * If both ``iconImage`` and ``iconClass`` are not available then the ``faSignInAlt`` from font awesome icons is used as icon image for SSO button.

Reviewer: Backend doesn't allow both to be empty. One of them will always be there. 
```suggestion
   * If ``iconClass`` doesn't match the supported icon classes then the ``faSignInAlt`` from font awesome icons is used as icon image for SSO button.
```
